Libs/MRML/Core/vtkMRMLStorageNode.cxx

```diff
@@ -395,6 +431,20 @@ void vtkMRMLStorageNode::PrintSelf(ostream& os, vtkIndent indent)
     (this->GetDefaultWriteFileExtension() ? this->GetDefaultWriteFileExtension() : "(none)") << "\n";
 
   os << indent << "TempFileName: " << (this->TempFileName ? this->TempFileName : "(none)") << "\n";
+  if (this->UserMessages)
+    {
+    os << indent << "UserMessages: " << &this->UserMessages << "\n";
+    for (int i = 0; i < this->UserMessages->GetNumberOfMessages(); i++)
+      {
+      vtkMessageCollection::Message message = this->UserMessages->GetMessage(i);
+      os << indent << "UserMessagesMember: (" << message.MessageType << ") "
```

Comment:
To be consistent with how `FileListMember` and `URIListMember` are displayed, what about removing the parenthesis ?

Reply:
Will do.